torch/_dynamo/symbolic_convert.py

```diff
@@ -1491,6 +1493,16 @@ def MATCH_KEYS(self, inst):
     def RESUME(self, inst):
         pass
 
+    def BINARY_OP(self, inst):
+        # linter complains about dis missing _nb_ops in py < 3.11
```

Comment:
Is it acutally there? Should we just silence the linter to make this work on all python versions?

Reply:
`_nb_ops` is not present in python < 3.11

Reply:
Ho so the comment needs updating right? The attribute is not there for py < 3.11. It has nothing to do with the linter.